src/article.js

```diff
@@ -487,3 +487,35 @@ Article.prototype.handleResize_ = function() {
     this.sections[i].rerender();
   }
 };
+
+/**
+ * Private method
+ * Removes blank paragraphs components from direction start/end
+ * @param {object} `component` specifiy the start from component.
+ * @param {boolean} `end` detrimines if direction is end.
+ */
+function _trimDirection(component, end) {
+  if (component instanceof Paragraph && component.trimTextTest()) {
+    var recursionComponent = component['get' + (end ? 'Previous' : 'Next') + 'Component']();
```

Comment:
I'd rewrite this with a normal if else to make it a little bit more readable, as well as make sure we always have max 80 characters per line.